sdk/lib/atl/atlbase.h

```diff
@@ -662,6 +665,127 @@ class CAtlDllModuleT : public CAtlModuleT<T>
     }
 };
 
+
+template <class T>
+class CAtlExeModuleT : public CAtlModuleT<T>
+{
+public:
+    DWORD m_dwMainThreadID;
+    //DWORD m_dwTimeOut;
+    //DWORD m_dwPause;
+    //bool m_bDelayShutdown;
+
+    CAtlExeModuleT()
```

Comment:
Do we usually put the exception "qualifiers" (like throw() ) at the end of member functions? If so we may say that both the ctor and dtor have throw()

Reply:
ditto